examples/apps/spaces/src/fluid-object/spacesItemMap.ts

```diff
@@ -37,10 +40,34 @@ function createSingleHandleItem(subFactory: IFluidDataStoreFactory) {
     };
```

Comment:
Is there a friendlier way to `await context.containerRuntime.createDataStore(packagePath)` ? :) I have seen this done in other places, just curious if there is a way to achieve this without reaching into runtime.

Reply:
I don't think we have a great pattern currently though I agree with you.  Probably a better alternative would create a data object that is unbound to any runtime and hand it back, for the caller of this function to bind to its own (private) runtime.